parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -209,6 +223,39 @@ ensureAPMZ() {
     ln -Ffs "$bin_path" "$apmz_filepath" # symlink apmz into /usr/local/bin/apmz
 }
 
+# install mostly-static bpftrace binary linked only against glibc > 2.23
+# it should be comptaible with any linux distro using a new enough kernel.
+# this may in the future require filtering the host OS if we encounter compatibility issues.
+# we also install the most commonly used bpftrace scripts by default.
+installBpftrace() {
+    local version=$1
```

Comment:
if the version isn't user configurable, why make it an input? Right now we have to update it in two places for version updates (the vhd script and cse main). Unless there are plans to make it configurable, I think it makes more sense to keep the version inside the install function.

Reply:
good point, fixed